gost.go

```diff
@@ -193,7 +193,7 @@ func fetchDeps(pkg string, branch string, update bool, alreadyFetched map[string
 		if dep == "" || dep == "." {
 			continue
 		}
-		if isGithub(dep) {
+		if isGithub(dep) || isBitBucket(dep) {
```

Comment:
Seems repetitive - probably want to factor these two checks into a single function like `supportsSubTrees` or something like that.